app/server/lib/DocSnapshots.ts

```diff
@@ -350,15 +350,26 @@ export function shouldKeepSnapshots(snapshots: ObjSnapshotWithMetadata[], snapsh
 
   // Get time of current version
   const start = moment.tz(current.lastModified, tz);
-
-  // Track saved version per hour, day, week, month, year, and number of times a version
-  // has been saved based on a corresponding rule.
-  const buckets: TimeBucket[] = [
-    {range: 'hour', prev: start, usage: 0, cap: 25},
-    {range: 'day', prev: start, usage: 0, cap: 32},
-    {range: 'isoWeek', prev: start, usage: 0, cap: 12},
-    {range: 'month', prev: start, usage: 0, cap: 96},
-    {range: 'year', prev: start, usage: 0, cap: 1000}
+ // Read environment variable for cap values as a stringified JSON object with key-value pairs
+    const capObjectString = process.env.GRIST_SNAPSHOT_TIMEBUCKET_CAP || '{"hour": "25", "day": "32", "isoWeek": "12", "month": "96", "year": "1000"}';
```

Comment:
Why `"hour": "25"` rather than `"hour": 25`?

Reply:
that's better. i'll update the code